First MVC: Validation: Clear old null values (#32743)
dotnet/AspNetCore.Docs: aspnetcore/tutorials/first-mvc-app/validation.md

The validation support provided by MVC and Entity Framework Core Code First is a good example of the DRY principle in action. You can declaratively specify validation rules in one place (in the model class) and the rules are enforced everywhere in the app.

## Delete the previously edited data

> In the next step validation rules are added that will no longer allow null values. 

Reviewer: ```suggestion
In the next step, validation rules are added that don't allow null values. 
```

> Run the app, navigate to `/Movies/Index`, delete all listed movies and stop the app.  The app will use the seed data the next time it is run.

Reviewer: ```suggestion
Run the app, navigate to `/Movies/Index`, delete all listed movies, and stop the app.  The app will use the seed data the next time it is run.
```
